.spelling

```diff
@@ -327,6 +328,7 @@ dereference
 dereferenced
 deserialize
 deserializing
+DestinationRule
```

Comment:
[citation needed]

I'm fine with that applying to the documentation, but not to the blog.  (And I am willing to die on this hill.)

Reply:
OK, upon reflection I'm less concerned, I'm happy to avoid this if it makes it easier to catch typos in the docs. (Note, VirtualService was in here already.)